src/app/tests/suites/certification/PICS.yaml

```diff
@@ -2735,18 +2735,64 @@ PICS:
     - label: "Does the DUT(server) support the ExpiringUserTimeOut attribute?"
       id: DRLK.S.A0035
 
+    - label:
+          "Does the DUT(server) support the AliroReaderVerificationKey attribute
+          and is ReadOnly"
```

Comment:
What does ReadOnly have to do with this PICS?

Same for the other ones here.

Reply:
Nothing. its unnecessary, will fix this one.